superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -133,6 +138,27 @@ const ResultSetErrorMessage = styled.div`
   padding-top: ${({ theme }) => 4 * theme.gridUnit}px;
 `;
 
+const updateDataset = async (
+  datasetId: number,
+  sql: string,
+  columns: Array<Record<string, any>>,
+  overrideColumns: boolean,
+) => {
+  const endpoint = `api/v1/dataset/${datasetId}?override_columns=${overrideColumns}`;
```

Comment:
We no longer need to pass `override_columns`, right? If so, let's remove it from the API and from here.

Reply:
We still need it to bypass the column uniqueness check:
https://github.com/apache/superset/blob/ac1d779a30ea8b009007b02b2bbb7cbe254ec16c/superset/datasets/commands/update.py#L138